source/main/gui/panels/GUI_MainSelector.cpp

```diff
@@ -887,7 +888,17 @@ void CLASS::SetPreviewImage(Ogre::String texture)
 void CLASS::ResizePreviewImage()
 {
 	MyGUI::IntSize imgSize(0, 0);
-	Ogre::TexturePtr t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	Ogre::TexturePtr t;
+	try
+	{
+		t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	}
+	catch (...)
+	{
+		Ogre::LogManager::getSingleton().stream() << "[RoR|SelectorGUI] Failed to load preview image: " << m_preview_image_texture;
+		m_preview_image_texture.clear();
+		m_Preview->setVisible(false);
+	}
```

Comment:
I would throw this `Ogre::LogManager::getSingleton().stream() << "[RoR|SelectorGUI] Failed to load preview image: " << m_preview_image_texture;` into the catch block inside of `SetPreviewImage(Ogre::String texture)` and call it a day.